src/rust/ide/src/model/execution_context.rs

```diff
@@ -20,6 +21,14 @@ use uuid::Uuid;
 
 
 
+// =================
+// === Constants ===
+// =================
+
+const DATAFLOW_ERROR_MESSAGE:&str = "Dataflow error.";
```

Comment:
this should have docs telling when this error message will be used. For me its not clear now.